layouts/ListLayout.js

```diff
@@ -81,6 +84,17 @@ export default function ListLayout({ posts, title }) {
           })}
         </ul>
       </div>
+      {router.pathname.indexOf('/blog') > -1 && (
```

Comment:
I still think it is better to check on the pagination prop to decide whether to render this component rather than the router path. This makes it more flexible for someone who wants to also add pagination for tags or not even have pagination for blogs by not passing in the pagination prop